docs/source/app_developers_guide/about_event_subscriptions.rst

```diff
@@ -0,0 +1,137 @@
+**************************
+About Events Subscriptions
+**************************
+
+An event subscription consists of an event type (the name of the event) and an
+optional list of filters for this event type.
+
+An event is “in a subscription” if the event’s ``event_type`` field matches the
+subscription’s ``event_type`` field, and the event matches any filters that are
+in the event subscription. If multiple filters are included in a subscription,
+only events that pass *all* filters will be received.
+
+
+Event Filters
+=============
+
+An event filter specifies which event attributes to match. A filter has the
+following contents:
+
+  * A string describing the filter (the filter name)
+
+  * The filter type (simple or regular expression) and match type (any or all)
+
+An event filter consists of a key and a "match string". Filters apply only to
+the event attributes that have the same key as the filter. The filter's match
+string is compared against the attribute's value according to the type of
+event filter.
+
+The event filter type can either be ``SIMPLE`` (an exact match to the specified
+string) or ``REGEX`` (a match using a regular expression). In addition, the
+filter type can specify ``ANY`` (match one or more) or ``ALL`` (must match all).
+The following event filters are supported:
+
+* ``SIMPLE_ANY``:
+   The filter's "match string" must match at least one of the
+   event's attribute values. For example, this filter type with the match string
+   "``abc``" would succeed for an event with the following attributes, because
+   it matches the attribute value ``abc``.
+
+   .. code-block:: none
+
+      Attribute(key="address",value="abc")
+      Attribute(key="address",value="def")
+
+* ``SIMPLE_ALL``:
+   The filter's "match string" must match all of the event's
+   attribute values. This filter type with the match string "``abc``"
+   would fail with the previous example, because it does not match all the
+   attribute values.
+
+* ``REGEX_ANY``:
+   The filter's regular expression must evaluate to a match for
+   one or more of the event's attribute values. For example, this filter type
+   with the match string "``[ad][be][cf]``" would succeed for an event with the
+   following attributes, because it matches the attribute values ``abc`` and
+   ``abbbc``.
+
+   .. code-block:: none
+
+      Attribute(key="address",value="abc")
+      Attribute(key="address",value="abbbc")
+      Attribute(key="address",value="def")
+
+* ``REGEX_ALL``:
+   The filter's regular expression must evaluate to a match for
+   all of the event's attribute values. This filter type with the match string
+   "``[ad][be][cf]``" would fail with the previous example, because it doesn't
+   match all three attribute values. The match string ``[ad][be]*[cf]`` would
+   succeed.
+
+
+Event Subscription Protobuf
+===========================
+
+Event subscriptions are submitted and serviced over a ZMQ socket as messages
+using the protocol buffers format.
```

Comment:
I would say "over a ZMQ socket using the Validator's messaging protocol" since it is more complex than just protobuf